rl_algorithms/distillation/dqn_agent.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+"""DQN agent for episodic tasks in OpenAI Gym.
+
+- Author: Kyunghwan Kim
+- Contact: kh.kim@medipixel.io
+- Paper: https://storage.googleapis.com/deepmind-media/dqn/DQNNaturePaper.pdf (DQN)
+         https://arxiv.org/pdf/1509.06461.pdf (Double DQN)
+         https://arxiv.org/pdf/1511.05952.pdf (PER)
+         https://arxiv.org/pdf/1511.06581.pdf (Dueling)
+         https://arxiv.org/pdf/1706.10295.pdf (NoisyNet)
+         https://arxiv.org/pdf/1707.06887.pdf (C51)
+         https://arxiv.org/pdf/1710.02298.pdf (Rainbow)
+         https://arxiv.org/pdf/1806.06923.pdf (IQN)
+"""
+
+import os
+from typing import Tuple
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+import wandb
+
+from rl_algorithms.common.buffer.distillation_buffer import DistillationBuffer
+from rl_algorithms.dqn.agent import DQNAgent
+from rl_algorithms.registry import AGENTS, build_learner
+
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+
+
+@AGENTS.register_module
+class DistillationDQN(DQNAgent):
+    """DQN for policy distillation."""
+
+    # pylint: disable=attribute-defined-outside-init
+    def _initialize(self):
+        """Initialize non-common things."""
+        self.softmax_tau = 0.01
+        self.learner = build_learner(self.learner_cfg)
+
+        self.buffer_path = (
+            f"./distillation_buffer/{self.log_cfg.env_name}/"
```

Comment:
How about to change path?

`./distillation_buffer -> ./data/distillation_buffer`